selfdrive/car/tests/test_models.py

```diff
@@ -214,10 +214,10 @@ def test_panda_safety_carstate(self):
 
     # warm up pass, as initial states may be different
     for can in self.can_msgs[:300]:
+      self.CI.update(CC, (can.as_builder().to_bytes(), ))
       for msg in can_capnp_to_can_list(can.can, src_filter=range(64)):
         to_send = package_can_msg(msg)
         self.safety.safety_rx_hook(to_send)
-        self.CI.update(CC, (can_list_to_can_capnp([msg, ]), ))
```

Comment:
@adeebshihadeh do you remember why we passed in a single CAN message at a time here?

Reply:
don't think it's important for the warm up pass